Package.swift

```diff
@@ -0,0 +1,28 @@
+// swift-tools-version:5.2
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "JSONWebToken",
+    platforms: [
+        .iOS(.v12), .tvOS(.v12)
+    ],
+    products: [
+        // Products define the executables and libraries produced by a package, and make them visible to other packages.
+        .library(
+            name: "JSONWebToken",
+            targets: ["JSONWebToken"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+    ],
+    targets: [
+        // Targets are the basic building blocks of a package. A target can define a module or a test suite.
+        // Targets can depend on other targets in this package, and on products in packages which this package depends on.
+        .target(
+            name: "JSONWebToken",
+            dependencies: []),
+    ]
```

Comment:
Don't you want to add the test target, in order for people to see how to use the package ?

Reply:
the test target needs a container app with an entitlement because the library use a keychain hack to create keys, and I don't know how to do that with SPM. 
I'm working on a solution without the keychain hack and CryptoKit and I'll add a test target on the package